js/src/apbct-public--2--public.js

```diff
@@ -29,7 +29,9 @@ var ctMouseReadInterval = setInterval(function(){
 
 //Writting interval
 var ctMouseWriteDataInterval = setInterval(function(){
-	ctSetCookie("ct_pointer_data", JSON.stringify(ctMouseData));
+	if (ctPublic.data__key_is_ok) {
```

Comment:
It would be better not to init these intervals completely if the key is not checked yet.